robotiq_hardware_tests/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# See https://docs.ros.org/en/foxy/How-To-Guides/Ament-CMake-Documentation.html
+
+cmake_minimum_required(VERSION 3.8)
+project(robotiq_hardware_tests VERSION 0.0.1 LANGUAGES CXX)
```

Comment:
This VERSION number should be set in the package.xml and IIRC it should match the other versions else the ros release tools complain.

Reply:
Removed.